File: Java/LCM_HCF.java
package com.lev;

import java.util.Scanner;

public class LCM_HCF {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.print("Enter the numbers : ");
        int n1 = input.nextInt();
        int n2 = input.nextInt();
        int a = n1;
        int b = n2;
        int temp;
        int hcf;
        int lcm;
        while (b != 0){
            temp = b;
            b = a % b;
            a = temp;
        }
        hcf = a;
        lcm = n1 * n2 / hcf;
        System.out.println("LCM : "+lcm);
        System.out.println("HCF : "+hcf);
    }
}

File: Python/LCM_HCF.py
print("Enter two numbers: ",end='')
n1=int(input())
n2=int(input())
a=n1
b=n2
while(b!=0):
    temp=b
    b=a%b
    a=temp
hcf=a
lcm=n1*n2//hcf
print("LCM of",n1,n2,":",lcm)
print("HCF of",n1,n2,":",hcf)
